cmd/pitr/pxc/pxc.go

```diff
@@ -189,6 +189,31 @@ func (p *PXC) GetBinLogFirstTimestamp(ctx context.Context, binlog string) (strin
 	return timestamp, nil
 }
 
+// GetBinLogLastTimestamp return binary log file last timestamp
+func (p *PXC) GetBinLogLastTimestamp(ctx context.Context, binlog string) (string, error) {
+	var existFunc string
+	nameRow := p.db.QueryRowContext(ctx, "select name from mysql.func where name='get_last_record_timestamp_by_binlog'")
```

Comment:
One more minor one, for consistency's sake, maybe capitalize SQL keywords in the query.